plutus-core/plutus-core/src/PlutusCore/Parser/Builtin.hs

```diff
@@ -7,32 +7,34 @@ module PlutusCore.Parser.Builtin where
 import Data.Text qualified as T
 import Data.Text.Internal.Read (hexDigitToInt)
 import PlutusPrelude (Word8)
-import Text.Megaparsec (MonadParsec (takeWhileP), choice, many, manyTill)
+import Text.Megaparsec (MonadParsec (takeWhileP), choice, customFailure, getSourcePos, many, manyTill)
 import Text.Megaparsec.Char (char, hexDigitChar)
 import Text.Megaparsec.Char.Lexer qualified as Lex hiding (hexadecimal)
 
 import Data.ByteString (pack)
 import Data.Map (Map, empty, insert, lookup)
```

Comment:
Oh, and yeah, please use `Data.Map.Strict` and the strict versions of containers in general, unless there's a specific reason to a lazy one.